metaflow/plugins/cards/card_client.py

```diff
@@ -0,0 +1,191 @@
+from metaflow.datastore import DATASTORES, FlowDataStore
+from .card_resolver import resolve_paths_from_task, resumed_info
+from .card_datastore import CardDatastore
+from .exception import UnresolvableDatastoreException
+import os
+import tempfile
+
+
+class Card:
+    def __init__(
+        self,
+        card_ds,
+        type,
+        path,
+        html=None,
+        created_on=None,
+        from_resumed=False,
+        origin_pathspec=None,
+    ):
+        # private attributes
+        self._path = path
+        self._html = html
+        self._created_on = created_on
+        self._card_ds = card_ds
+
+        # public attributes
+        self.type = type
+        self.from_resumed = from_resumed
+        self.origin_pathspec = origin_pathspec
+
+        # Tempfile to open stuff in browser
+        self._temp_file = None
+
+    @property
+    def html(self):
```

Comment:
change this to `get()` to make it consistent with the CLI. Also it is polite to keep properties as no-cost, side-effect-free operations, whereas this call actually makes a network request etc. - so let's drop `@property` on this one